client/hooks/feed/index.js

```diff
@@ -1,6 +1,6 @@
 import { usePublicFeed } from './publicFeed';
 import { useUserPacks } from './../packs';
-import { useUserTrips } from './../trips';
```

Comment:
Why this change ?

Reply:
The trpc query hook was for singleTripsStore but was saved in a folder called trips instead of singleTrips, which was stopping me from saving my own trpc hook (which was the actually trpc trips hook) in a trips folder. I don't know if my explanation makes sense to you?

Reply:
i see what you are saying, but user trips will return more than 1 trip, so i think we should name it something else ? or maybe i'm misunderstanding

Reply:
No it wouldn't. All I did was rename the folder, so I can use the previous folder name for mine instead. There are no double files or double calls of `useUserTrips`. It's still the exact same folder/file structure, just with a different folder name. (from trips -> singletrips)

Reply:
Got it thanks for clarifying